docs/cli.md

```diff
@@ -1,183 +1,348 @@
-# NEO-GO client
-
-The Neo-Go client is used to run node, create/compile/deploy/invoke/debug smart contracts, run vm and operate with the wallet. 
+# NeoGo CLI interface
+
+NeoGo CLI provides all functionality from one binary, so it's used to run
+node, create/compile/deploy/invoke/debug smart contracts, run vm and operate
+with the wallet. The standard setup assumes that you're running a node as a
+separate process and it doesn't provide any CLI of its own, instead it just
+makes RPC interface available for you. To perform any actions you invoke NeoGo
+as a client that connects to this RPC node and does things you want it to do
+(like transferring some NEP-17 asset).
+
+All CLI commands have corresponding help messages, use `--help` option to get
+them, for example:
+```
+./bin/neo-go db --help
+```
 
 ## Running node
 
-To start Neo Go node locally it's required to run 
-[neo-privatenet](https://hub.docker.com/r/cityofzion/neo-privatenet/) Docker image on your machine.
+Use `node` command to run a NeoGo node, it will be configured using a YAML
+file that contains network parameters as well as node settings.
 
-To run node you can use
-```
-make run
-```
+### Configuration
+
+All config files are located in `./config` and they are differentiated according to the network type:
+- `protocol.mainnet.yml` belongs to `--mainnet` network mode (`-m` short option)
+- `protocol.privnet.yml` belongs to `--privnet` network mode (`-p` short
+  option) and is used by default
+- `protocol.testnet.yml` belongs to `--testnet` network mode (`-t` short option)
+- `protocol.unit_testnet.yml` used by unit tests
+
+If you want to use some non-default configuration directory path, specify
+`--config-path` flag:
+
+`./bin/neo-go node --config-path /user/yourConfigPath`
+
+The file loaded is chosen automatically depending on network mode flag.
 
-or run the binary directly:
+### Starting a node
+
+To start Neo node on private network use:
 
 ```
 ./bin/neo-go node
 ```
 
-By default the node will run on the `private network`, to change this use additional network flag:
+Or specify a different network with appropriate flag like this:
 
 ```
 ./bin/neo-go node --mainnet
 ```
 
-#### Available network flags
-- `--mainnet, -m`
-- `--privnet, -p`
-- `--testnet, -t`
+By default the node will run in foreground using current standard output for
+logging.
 
-#### Configuration path
+### DB import/exports
 
-If you want to use some specific configuration path you have to use `--config-path` flag
+Node operates using some database as a backend to store blockchain data. NeoGo
```

Comment:
Add some notes about db settings in config (like path is `./chains` by default)?

Reply:
#1849